src/data/set/basic.lean

```diff
@@ -195,7 +195,7 @@ nevertheless be useful for various reasons, e.g. to apply further projection not
 argument to `simp`. -/
 lemma _root_.has_mem.mem.out {p : α → Prop} {a : α} (h : a ∈ {x | p x}) : p a := h
 
-theorem nmem_set_of_eq {a : α} {p : α → Prop} : a ∉ {x | p x} = ¬ p a := rfl
+theorem nmem_set_of_iff_not {a : α} {p : α → Prop} : a ∉ {x | p x} ↔ ¬ p a := iff.rfl
```

Comment:
```suggestion
theorem nmem_set_of_iff {a : α} {p : α → Prop} : a ∉ {x | p x} ↔ ¬ p a := iff.rfl
```

Reply:
What is the guidance for when a lemma ends in `_iff` versus otherwise?

Reply:
I was half thinking of the same remark, but I decided that it's not such a priority since we don't have a consistent rule about naming it `set.mem_whatever` or `set.mem_whatever_iff` or `set.mem_whatever_iff_not`.

Reply:
I am happy to make the change, but indeed to me it seems much of a muchness as if there *is* a general rule I can't discern it.